Pipfile

```diff
@@ -58,6 +63,9 @@ sphinx_rtd_theme = "*"
 aafigure = "*"
 # CLI
 nucypher = {editable = true,path = "."}
+# UI testing
+pytest-dash = "==2.1.0"
+instapy-chromedriver = "*"
```

Comment:
Also here - Is this used for installing chromedriver?  Is this the recommended install?

Reply:
It is used for installing the chromedriver - it handles platform-specific installation of the driver.